### DISCUSSION TITLE
Styles not loading with MIME error after updating middleware (12.2)

### DISCUSSION THREAD

**Discussion by MerlinMason** [vercel/next.js#38208] [Help] [answered]
I've just updated our app to 12.2 and refactored the middleware in lines with the migration guide. Everything works fine locally but when deployed the styles won't load.

```
Refused to apply style from '<siteurl>' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```

We're using Tailwind so the only styles are imported with `import "tailwindcss/tailwind.css";` in `_app.tsx`. 

I'm quite confused by this - is the middleware somehow intercepting the loading request for the CSS? Any ideas?

**Comment by MerlinMason** [comment, marked as answer]
Update 🎉 

So... The middleware was doing some auth checking and redirecting users who are not logged in, this runs on all requests and was redirecting the loading of assets to a `/session-expired` page which didn't as an HTML page is not a CSS file.

Adding
```js
if (request.nextUrl.pathname.startsWith("/_next/")) {
    return true;
}
```
To the top of the middleware function seemed to do the trick.

**Reply by Lucasmm016** [reply]
The best thing to do is to return "return NextResponse.next()" instead of "return true"!
